src/pages/docs/platform-engineering/managing-runbook-resources.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2023-11-09
+modDate: 2023-11-09
+title: Managing runbook resources
+description: This section documents how to manage runbooks as code.
+navOrder: 6
+---
+
+[Runbooks](/docs/runbooks) are a component of a project, sharing or referencing much of the project's configuration such as variables, connected library variable sets, tenants, and lifecycles.
+
+However, it can be useful to treat runbooks as an independently deployable artifact. This allows a common runbook to be shared across many projects.
+
+Runbooks can be defined as a Terraform module and applied to an existing project, effectively "deploying" the runbook into the project.
+
+:::div{.hint}
+Runbooks are not managed by Config-as-code.
+:::
+
+Runbooks can be defined in a Terraform module in two ways:
+
+* Write the module by hand
+* Serialize an existing project to a Terraform module with [octoterra](https://github.com/OctopusSolutionsEngineering/OctopusTerraformExport)
+
+## Writing by hand
+
+The process of defining a runbook in Terraform is much the same as defining a project. Both a runbook and a project have the concept of a deployment process that defines the steps to be run. See the [Managing project resources](managing-project-resources) section for more information on defining steps in Terraform by hand.
```

Comment:
I _think_ this is the correct syntax (not 100% sure though :smile:) 
```suggestion
The process of defining a runbook in Terraform is much the same as defining a project. Both a runbook and a project have the concept of a deployment process that defines the steps to be run. See the [Managing project resources](/docs/platform-engineering/managing-project-resources) section for more information on defining steps in Terraform by hand.
```

Reply:
Either syntax appears to work. This is a screenshot of a published page, and it uses the relative path for a sibling page, which publishes the correct link when the markdown is compiled.

![image](https://github.com/OctopusDeploy/docs/assets/160104/35b71721-9e7b-4a6a-a80d-a6344974bc20)

Reply:
I'll stick with the simpler syntax for now as that it how the other pages define links between themselves.

Reply:
Thanks for checking/confirming - good to know!